cern_access/indico_cern_access/migrations/20241015_1702_92377810f14e_modify_reservation_code_field.py

```diff
@@ -0,0 +1,29 @@
+"""Modify reservation code field
```

Comment:
That does not describe what the revision does, better use something like "Make reservation code unique" (also rename the file accordingly).